js/src/hooks/usePolicyCheck.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Internal dependencies
+ */
+import useAppSelectDispatch from './useAppSelectDispatch';
+
+/**
+ * Get policy detail info from calling `useAppSelectDispatch` with `getPolicyCheck`.
+ * Returns `{ hasFinishedResolution, data, invalidateResolution }`.
+ *
+ * `data` is an object of policy check mapping. e.g.:
+ *
+ * ```json
+ * {
+ *	'policy_check' =>	[
+ *				'allowed_countries'	=> true,
+ *                              'robots_restriction'	=> false,
+ *                              'page_not_found_error'	=> false,
+ *                              'page_restricts'	=> false,
+ *				'store_ssl'		=> true,
+ *				'payment_gateways'	=> true,
+ *				'refund_returns'	=> true,
+ *				]
+ * }
+ * ```
```

Comment:
The indentations are mixed of spaces and tabs, and the syntax of this example is not a valid JSON format nor fits with the actual data structure. (`data` should be a flat object without the `policy_check` layer.)

Reply:
Updated.